recipes/hmtnote/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hmtnote" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 843300a160a0947ab3572c220198e88cc5d6b0fdf554f3a168e9cabdc6e66e8e
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - hmtnote=hmtnote.cli:main
+  script: sed -i.bak "s/pytest-runner//g" setup.py; {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
+
+requirements:
+  host:
+    - aiofiles ==0.4.0
```

Comment:
It really needs these exact versions?

Reply:
I have now replaced with >= version no as it is minimum requirement